Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Revolvers/anaconda.yml

```
@@ -30,9 +30,9 @@
       path: /Audio/_Impstation/Weapons/Guns/Gunshots/anaconda.ogg
   - type: RevolverAmmoProvider
     whitelist:
-      tags:
-        - CartridgeAntiMateriel
-        - SpeedLoaderAM
+    tags:
+    - CartridgeAntiMateriel
+    - SpeedLoaderAM
```

One indent too much removed here
```suggestion
    whitelist:
      tags:
      - CartridgeLoaderAntiMateriel
      - SpeedLoaderAM
```